packages/table/src/table.scss

```diff
@@ -163,3 +163,70 @@ $menu-z-index: $row-z-index + 1 !default;
   height: $pt-grid-size * 10;
   overflow: scroll;
 }
+
+// TEMPORARY
```

Comment:
I guess most of these styles are actually not temporary; it's just the background colors that are.